scania-digital-design-system/sdds
# fix(dropdown): z-index, open/close + uuid

**Describe pull-request**  
Fixing issues in the dropdown component.

1. Dropdown z-index of menu items.
2. Dropdown not working when placed in the sdds-modal component


**Solving issue**  
Fixes: Bug reported through Teams channel and in a video call.

**How to test**  
1. Start the development environment from the root
3. run linking command 
4. run linking command in Angular12 demo example
5. run Angular12 example
6. See if dropdown menus looks broken when two dropdowns are close to each other
7. See if dropdown works when placed in the modal component

[DOCS: How to link](https://scaniaazureservices-my.sharepoint.com/:o:/r/personal/nikola_johnny_mirkovic_scania_com/Documents/Notebooks/IXCD%20Book?d=wa2c3d5d9d0dc4cb9bee55e8a81cedb1b&csf=1&web=1&e=Fhq7wB)
